weld/src/main/java/org/jboss/as/weld/ejb/SessionObjectReferenceImpl.java

```diff
@@ -37,15 +38,18 @@
 import org.jboss.weld.ejb.api.SessionObjectReference;
 
 /**
- * Implementation for non-stateful beans, a new view instance is looked up each time
+ * Implementation for non-stateful beans.
  *
  * @author Stuart Douglas
  */
 public class SessionObjectReferenceImpl implements SessionObjectReference {
 
     private final Map<String, ServiceName> viewServices;
+
     private final String ejbName;
 
+    private transient volatile Map<String, ManagedReference> businessInterfaceToReference;
```

Comment:
It looks like this is only accessed in a synchronized method, so I don't think this needs to be volatile